docs/examples/auto-instrumentation/README.rst

```diff
@@ -153,7 +153,7 @@ and run the following command instead:
 
 .. code:: sh
 
-    $ opentelemetry-instrument --trace-exporter console_span python server_uninstrumented.py
+    $ opentelemetry-instrument --traces_exporter console python server_uninstrumented.py
```

Comment:
unrelated to the fix in this PR

Reply:
> Also updates auto-instrumentation doc and add some type hints.

Not in the title but in the description.